kythe/extractors/BUILD

```diff
@@ -81,7 +81,7 @@ extractor_action(
     ],
     data = ["//external:vnames_config"],
     extractor = "@io_kythe//kythe/go/extractors/cmd/bazel:bazel_go_extractor",
-    mnemonics = ["GoCompile"],
+    mnemonics = ["GoCompilePkg"],
```

Comment:
Mention that we only support Bazel rules_go 0.19.0 and up.

Reply:
done